### PR TITLE
contrib/lite: spelling/code tweaks

### PR DESCRIPTION
**Pull request by brettkoonce**

### CODE REVIEW

**Review by andrehentz — APPROVED** [review]

**Comment by brettkoonce** [comment]
will fix the test in a sec

**Comment by brettkoonce** [comment]
fixed test, rebased to match upstream

File: tensorflow/contrib/lite/interpreter.cc [hunk 1]
```diff
@@ -182,7 +182,12 @@ TfLiteStatus Interpreter::ReplaceSubgraphsWithDelegateKernels(
 
   execution_plan_.clear();
   for (auto& subgraph : subgraphs) {
-    // Subgraphs calimed by the delegate should have a "macro" op created, the
+    // Turn subgraph.nodes into a TfLiteIntArray compatible data structure.
```
Comment (brettkoonce) [inline tensorflow/contrib/lite/interpreter.cc]: should I be rebasing on a different branch?

**Comment by brettkoonce** [comment]
@gunan removed issue introduced by rebase, please run test again 🎉

**Pull request merged**